src/Common/src/Interop/OSX/Interop.Libraries.cs

```diff
@@ -11,6 +11,7 @@ private static partial class Libraries
         internal const string libproc = "libproc";
         internal const string LibSystemCommonCrypto = "/usr/lib/system/libcommonCrypto";
         internal const string LibSystemKernel = "/usr/lib/system/libsystem_kernel";
+        internal const string Odbc32 = "libodbc.2.dylib";
```

Comment:
Once you add it into the Unix file you can delete this one.

Reply:
My mistake. It's supposed to be different for linux and that's why I added the new file